pandas/tests/extension/test_numpy.py

```diff
@@ -55,6 +55,25 @@ def _can_hold_element_patched(obj, element) -> bool:
     return can_hold_element(obj, element)
 
 
+orig_assert_attr_equal = tm.assert_attr_equal
+
+
+def _assert_attr_equal(attr: str, left, right, obj: str = "Attributes"):
+    """
```

Comment:
umm can u avoid doing this ; eg why not just fix our method itself?

Reply:
This comes down to #40021, we've mostly landed on patching here rather than in non-test code